tests/redirects_tests/tests.py

```diff
@@ -86,3 +86,15 @@ def test_response_redirect_class(self):
         Redirect.objects.create(site=self.site, old_path='/initial/', new_path='/new_target/')
         response = self.client.get('/initial/')
         self.assertEqual(response.status_code, 302)
+
+
+class DisallowedRedirectTests(TestCase):
+    def test_redirect_formatting(self):
+        r = http.HttpResponseRedirect.__new__(http.HttpResponseRedirect)
+        with self.assertRaises(DisallowedRedirect):
+            http.HttpResponseRedirect.__init__(r, "ssh://foo")
+
+        # Check that formatting an invalid redirect does not crash
```

Comment:
Use a docstring that states the expected behavior. "Check that" prefix not needed.